Added air-gap install steps with install.sh (#2005)
rancher/rke2: docs/install/airgap.md

```

**Note:** The `system-default-registry` parameter must specify only valid RFC 3986 URI authorities, i.e. a host and optional port.

### RKE2 Install.sh Script Install

> You may use the `install.sh script` and define the `INSTALL_RKE2_ARTIFACT_PATH` to reduce steps and create the systemd service file.

Reviewer: ```suggestion
`install.sh` may be used in an offline mode by setting the `INSTALL_RKE2_ARTIFACT_PATH` variable to a path containing pre-downloaded artifacts. This will run though a normal install, including creating systemd units.
```

> 1. Download the rke2, rke2-images, and sha256sum archives from the release into a directory, as in the example below:

Reviewer: ```suggestion
1. Download the install script, rke2, rke2-images, and sha256sum archives from the release into a directory, as in the example below:
```

> ```
> # mkdir /root/rke2-artifacts && cd /root/rke2-artifacts/
> # curl -OLs https://github.com/rancher/rke2/releases/download/v1.21.5%2Brke2r2/rke2-images.linux-amd64.tar.zst
> # curl -OLs https://github.com/rancher/rke2/releases/download/v1.21.5%2Brke2r2/rke2.linux-amd64.tar.gz
> # curl -OLs https://github.com/rancher/rke2/releases/download/v1.21.5%2Brke2r2/sha256sum-amd64.txt
> # curl -sfL https://get.rke2.io --output install.sh
> ```

Reviewer: Just include the commands to run, and without a # prefix so that users can just copy-paste the commands.
```suggestion
```bash
mkdir /root/rke2-artifacts && cd /root/rke2-artifacts/
curl -OLs https://github.com/rancher/rke2/releases/download/v1.21.5%2Brke2r2/rke2-images.linux-amd64.tar.zst
curl -OLs https://github.com/rancher/rke2/releases/download/v1.21.5%2Brke2r2/rke2.linux-amd64.tar.gz
curl -OLs https://github.com/rancher/rke2/releases/download/v1.21.5%2Brke2r2/sha256sum-amd64.txt
curl -sfL https://get.rke2.io --output install.sh
```
```

2. Next, run install.sh using the directory, as in the example below:

> ``` 
> # chmod +x install.sh
> # INSTALL_RKE2_ARTIFACT_PATH=/root/rke2-artifacts ./install.sh
> [INFO]  staging local checksums from /root/rke2-artifacts/sha256sum-amd64.txt
> [INFO]  staging zst airgap image tarball from /root/rke2-artifacts/rke2-images.linux-amd64.tar.zst
> [INFO]  staging tarball from /root/rke2-artifacts/rke2.linux-amd64.tar.gz
> [INFO]  verifying airgap tarball
> grep: /tmp/rke2-install.nc03lrohgR/rke2-images.checksums: No such file or directory
> [INFO]  installing airgap tarball to /var/lib/rancher/rke2/agent/images
> [INFO]  verifying tarball
> [INFO]  unpacking tarball file to /usr/local
> ```

Reviewer: Just include the commands to run, and without a # prefix so that users can just copy-paste the command.
```suggestion
```bash
INSTALL_RKE2_ARTIFACT_PATH=/root/rke2-artifacts sh install.sh
```
```
